CHANGELOG.md

```diff
@@ -9,6 +9,33 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Enhancements
 
+-
+
+### Fixes
+
+-
+
+### Internal Changes
+
+-
+
+### Deprecations
+
+-
+
+### Removals
+
+-
+
+### Security
+
+-
+
+
+## [0.1.50] - 2022-04-28
+
+### Enhancements
+
 - The `api-resource-collector` `ClusterRole` has been updated to fetch network
```

Comment:
Let's remove the `Deprecations`, `Removals`, and `Security` sections below since there isn't any relevant content for this release (similar to what we did before).